tests/functional/show/test_show.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from dbt.exceptions import DbtRuntimeError
+from dbt.tests.util import run_dbt_and_capture, run_dbt
+from tests.functional.show.fixtures import (
+    BaseConfigProject,
+    models__second_ephemeral_model,
+)
+
+
+class TestShow(BaseConfigProject):
+    def test_none(self, project):
+        with pytest.raises(
+            DbtRuntimeError, match="Either --select or --inline must be passed to show"
+        ):
+            run_dbt(["deps"])
```

Comment:
Do we have some project that doesn't need to run deps? It doesn't feel good that we need to run a deps for each testcase